src/network/networkctl.c

```diff
@@ -166,13 +166,10 @@ static int decode_link(sd_netlink_message *m, LinkInfo *info, char **patterns) {
                 sd_netlink_message_read_u32(m, IFLA_MTU, &info->mtu) >= 0 &&
                 info->mtu > 0;
 
-        info->has_min_mtu =
-                sd_netlink_message_read_u32(m, IFLA_MIN_MTU, &info->min_mtu) >= 0 &&
-                info->min_mtu > 0;
-
-        info->has_max_mtu =
-                sd_netlink_message_read_u32(m, IFLA_MAX_MTU, &info->max_mtu) >= 0 &&
-                info->min_mtu > 0;
+        if (info->has_mtu) {
```

Comment:
hmm these are different  properties so how can they be merged with has mtu ?

Reply:
Are there any possibility that no MTU but minimum or maximum?
Please comment on #12706. Thank you.

Reply:
Is n't it depend on the driver code. It may chose to implement or not.

Reply:
Theoretically yes. But I'd like to ignore such cases...